spec/controllers/curation_concerns/generic_works_controller_spec.rb

```diff
@@ -4,7 +4,16 @@
 # which is included into .internal_test_app/app/controllers/generic_works_controller.rb
 describe CurationConcerns::GenericWorksController do
   let(:user) { create(:user) }
-  before { sign_in user }
+  let(:work) { create(:generic_work, user: user) }
+  let(:sipity_entity) do
+    create(:sipity_entity,
+           proxy_for_global_id: work.to_global_id.to_s,
+           workflow_state_id: 2)
```

Comment:
As discussed in Slack, you should be able to :fire: all the hard-coded `workflow_state_id` stuff you added.